src/log/list.js

```diff
@@ -15,12 +15,14 @@ const { removeUndefinedKeys } = require('../utils/objUtils')
  * @param {string} params.processId
  * @param {number} params.limit
  * @param {number} params.startTime
- * @param {number} params.endTime
+ * @param {string} params.level
+ * @param {string} params.subject
+ * @param {number} params.skip
  * @returns {Promise<ApiResponse>}
  */
 
 module.exports = serializeAxiosResponse(
-  ({ limit, processId, startTime, endTime, deviceId, userId }) => (
+  ({ limit, processId, startTime, endTime, deviceId, userId, level, subject, skip }) => (
```

Comment:
The problem with this is a it already predefines which keys are available (and we are about to deprecate startTime and endTime) 

thats not the job of an sdk, neither is it to provide documenation for the endpoint

So ,I would propose a breaking change here, rather than destructuring each currently valid querystringparam we just have parameter `(queryStringParams={})`

Reply:
@FelixMZ2018  please write this in Readme and create a ticket to improve other similar places found

Reply:
checked queryStringParameters, this should be the only place its using this destructuring format, every other function should be agnostic already.